source/frontend/src/features/mapSideBar/acquisition/common/GenerateForm/hooks/useGenerateExpropriationForm9.test.tsx

```diff
@@ -117,16 +117,13 @@ describe('useGenerateExpropriationForm9 functions', () => {
   it(`sends "Expropriation vesting date" in the json payload to generation api`, async () => {
```

Comment:
same thing here, testing for the expropriation vesting date doesn't make sense anymore.